neqo-transport/src/recovery/sent.rs

```diff
@@ -174,6 +194,8 @@ pub struct SentPackets {
 }
 
 impl SentPackets {
+    #[allow(clippy::len_without_is_empty)]
```

Comment:
Is this the type you wanted to add an `is_empty()` function for?

Reply:
The lint only complains, because `SentPackets` is now `pub` when `bench` feature is enabled.

Given that we don't need `SentPackets::is_empty` anywhere within the `neqo_transport::recovery` `mod` (i.e. parent of `sent` `mod`), I think suppressing is the better option here.